README.md

```diff
@@ -1,17 +1,34 @@
-<h1 align="center">
-
-🌀 Portal.JS<br/>
-The javascript framework for<br/>
-data portals
-
+<h1 align="center">🌀 Portal.JS
+> The JavaScript framework for data portals
 </h1>
 
-🌀 `portal` is a framework for rapidly building rich data portal frontends using a modern frontend approach. `portal` can be used to present a single dataset or build a full-scale data catalog/portal.
+* [What is Portal.JS ?](#What-is-Portal.JS)
+    * [Features](#Features)
+      * [For developers](#For-developers)
+* [Installation and setup](#Installation-and-setup)
+* [Getting Started](#Getting-Started)
+* [Tutorial](#Tutorial)
+    * [Build a single Frictionless dataset portal](#Build-a-single-Frictionless-dataset-portal)
+    * [Build a CKAN powered dataset portal](#Build-a-CKAN-powered-dataset-portal)
+* [Architecture / Reference](#Architecture--Reference)
+    * [Component List](#Component-List)
+        * [UI Components](#UI-Components)
+        * [Dataset Components](#Dataset-Components)
+        * [View Components](#View-Components)
+        * [Search Components](#Search-Components)
+        * [Blog Components](#Blog-Components)
+        * [Misc Components](#Misc-Components)
+    * [Concepts and Terms](#Concepts-and-Terms)
+        * [Dataset](#Dataset)
+        * [Resource](#Resource)
+        * [View Spec](#view-spec)
+* [Appendix](#Appendix)
+    * [What happened to Recline?](#What-happened-to-Recline?)
 
-`portal` is built in Javascript and React on top of the popular [Next.js][] framework. `portal` assumes a "decoupled" approach where the frontend is a separate service from the backend and interacts with backend(s) via an API. It can be used with any backend and has out of the box support for [CKAN][].
+# What is Portal.JS
+🌀 `portal.js` is a framework for rapidly building rich data portal frontends using a modern frontend approach. `portal.js` can be used to present a single dataset or build a full-scale data catalog/portal.
 
-[ckan]: https://ckan.org/
```

Comment:
We prefer out of line links if possible as makes for cleaner reading and easier reuse. Very minor.